static/js/biomedical/disease/page.tsx

```diff
@@ -85,13 +90,33 @@ export class Page extends React.Component<PagePropType, PageStateType> {
 
     return (
       <>
-        <h2>Disease Browser</h2>
+        <h2>{diseaseName}</h2>
+        <h6>
+          <a href={diseaseLink}>Graph Browser View</a>
+        </h6>
         <h5>Disease-Gene Association</h5>
+        <p>
+          The association score of {diseaseName} with genes as reported by
+          DISEASES by Jensen Lab. Associations were determined by text mining of
```

Comment:
should we add a link for DISEASES, SPOKE and ChEMBL?

Reply:
That's a good idea! What do you think @spiekos ?

Reply:
After discussing it with Sam, we are thinking of adding the name and source of the provenance for each of the graphs. But, instead of hardcoding it, we're planning to getting it from the config.